src/components/ActivityForm.js

```diff
@@ -10,40 +11,40 @@ export default class ActivityForm extends React.Component {
       room: props.activity ? props.activity.room : '',
       teacher: props.activity ? props.activity.teacher : '',
       error: ''
-      //TODO: when you will use prop-types you will be able to improve this place ;)
+      //TODO: when you will use prop-types you will be able to improve this place ;)
```

Comment:
I think, that now (looking at previous comments, maybe you will be able to improve). If not, because `activity` will be able to be undefined, in separate PR you should check `lodash` package and `get` method ;)